### PR TITLE
add config export

### PR DESCRIPTION
**Pull request by danewalton**
This was missing the installation of the uhttp targets.

Brings in line with umqtt which is a bit of a better model citizen.

https://github.com/Azure/azure-umqtt-c/blob/09558f380a314ea89b28f406cc810f47718cea49/CMakeLists.txt#L155-L198

### CODE REVIEW

**Comment by momuno** [comment]
with my limited knowledge in the space, i think it looks good.

File: configs/uhttpConfig.cmake [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+include("${CMAKE_CURRENT_LIST_DIR}/uhttpTargets.cmake")
+
+get_target_property(UHTTP_INCLUDES uhttp INTERFACE_INCLUDE_DIRECTORIES)
+
+set(UHTTP_INCLUDES ${UHTTP_INCLUDES} CACHE INTERNAL "")
```
Comment (momuno) [inline configs/uhttpConfig.cmake:8]: Does this need an extra line or do we not care for cmake files?

**Review by momuno — APPROVED** [review]

File: configs/uhttpConfig.cmake (hunk 1 above)
Comment (danewalton) [reply configs/uhttpConfig.cmake]: yup newlines everywhere :)

**Comment by danewalton** [comment]
> with my limited knowledge in the space, i think it looks good.

thanks. yea this is just doing exactly what uMQTT is doing.

**Pull request merged**